backend/src/main.ts

```diff
@@ -203,14 +216,34 @@ export const moodLog = async (req: UserRequest, res: Response) => {
     let globalNow = DateTime.utc();
 
     if (data.addFlag) {
-        if (typeof data.addFlag !== "string" || !data.addFlag.startsWith("summary:")) {
+        if (typeof data.addFlag !== "string") {
+            console.log("Invalid addFlag:", data.addFlag);
             res.send(400);
             return;
         }
 
-        globalNow = DateTime.fromISO(data.addFlag.split(":")[1], { zone: data.timezone });
+        if (data.addFlag.startsWith("summary:")){
```

Comment:
```suggestion
        if (data.addFlag.startsWith("summary:")) {
```
get ur hand formatting game up